Tasks/app-store-promote/package.json

```diff
@@ -4,9 +4,14 @@
   "description": "Azure Pipelines task to promote a build in the Apple App Store",
   "main": "app-store-promote.js",
   "dependencies": {
-    "azure-pipelines-task-lib": "^2.11.1"
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
+    "@types/uuid": "^8.3.0"
```

Comment:
nit: do we need types for uuid here? I don't see this package being used anywhere